components/kyma-environment-broker/internal/avs/delegator.go

```diff
@@ -21,14 +20,14 @@ const (
 )
 
 type Delegator struct {
-	operationManager *process.OperationManager
+	operationStorage storage.Provisioning
```

Comment:
couldn't use `OperationManager` because of the imports loop with provisioning package

Reply:
It is strange, why only the AvS suffers? what is the cycle (which packages)?

Reply:
I'm not sure but probably because all steps have logic inside `process/provisioning` dir, and this step all logic has outside. In `process/provisioning` this step has only initiation.

Reply:
rename manger to (ProvisioningManager|DeprovisioningManager)

Reply:
reverted